modules/apps/27-interchain-accounts/types/packet.go

```diff
@@ -66,27 +66,32 @@ The Memo format is defined like so:
 {
 	// ... other memo fields we don't care about
 	"callbacks": {
-		"src_callback_address": {contractAddrOnSrcChain},
+		"src_callback_address": {contractAddrOnSourceChain},
 		"dest_callback_address": {contractAddrOnDestChain},
```

Comment:
This is nit, but if we spell out `Src` as `Source`, should we do the same with `Dst` and written `Destination` instead?

Reply:
I don't thinkso. We changed this to be consistent with existing [terminology](https://github.com/cosmos/ibc-go/blob/main/modules/core/exported/channel.go#L28). If we didn't have an existing precedence, it likely would have remained `src`